fossunited/fossunited/web_template/upcoming_foss_events_section/upcoming_foss_events_section.html

```diff
@@ -64,7 +57,7 @@ <h5>Upcoming Events</h5>
         </div>
 
         {% from "fossunited/templates/macros/event_card.html" import event_card %}
-        {% set events = frappe.get_all("FOSS Chapter Event", fields=["*"], filters={"status": ["in", ["Approved", "Live"]], "is_published": 1, "event_start_date": ['>=', frappe.utils.now()]}, page_length=9, order_by='event_start_date') %}
+        {% set events = frappe.get_all("FOSS Chapter Event", fields=["*"], filters={"status": ["in", ["Approved", "Live"]], "is_published": 1, "event_start_date": ['>=', frappe.utils.now()]}, page_length=4, order_by='event_start_date') %}
```

Comment:
why page_length only 4? We have too many monthly events for it to be only 4. Atleast 8 should be the number given we have a 4x4 grid situation

Reply:
IndiaFOSS card will be hidden if its 4

Reply:
I will make it 8 in page_length. Reasons why reduced to 4: 

- Our current scenario of upcoming events - number never went upto 8 at a point. A filled grid seemed good rather than a filled and half filled. 

- IndiaFOSS card was also present in the flagship events' space as well